src/app/src/main/kotlin/org/imperial/mrc/hint/controllers/BaselineController.kt

```diff
@@ -30,14 +32,18 @@ class BaselineController(private val fileManager: FileManager) {
         // for now just read basic file info from upload dir
         val file = fileManager.getFile(FileType.PJNZ)
 
-        return if (file != null) {
+        val data = if (file != null) {
             val fileName = file.name
             val countryName = fileName.split("_").first()
-            "{\"pjnz\": { \"filename\": \"$fileName\", \"country\": \"$countryName\"}}"
+            mapOf("pjnz" to buildPjnzResponse(fileName, countryName))
         } else {
-            "{\"pjnz\": null}"
+            mapOf("pjnz" to null)
         }
+        return SuccessResponse(data).toJsonString()
```

Comment:
Returning a response that conforms to the schema. This is still just a mock response until hintr is actually able to validate files.